.env.test

```diff
@@ -30,6 +30,7 @@ ENABLE_PROGRESS_GRAPH_SETTINGS=false
 ENABLE_TEAM_TYPE_SETTING=false
 ENABLE_NEW_EDITOR_PAGES=true
 ENABLE_UNIT_PAGE=true
+ENABLE_ASSETS_PAGE=true
```

Comment:
Actually values for .env.test is not being picked up by `getConfig` in tests probably due to `initialize` function not being called for tests. Since it is out of scope for this PR, I updated the env value to false in case these values are picked up in future.